.github/workflows/ci.yaml

```
@@ -0,0 +1,98 @@
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: ${{ !github.event.pull_request.draft }} # only run on non-draft PRs
+
+    steps:
+      - name: Checkout sdk-py
+        uses: actions/checkout@v4
+
+      - name: Set up Python
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)